include/openmc/plot.h

```diff
@@ -52,7 +52,30 @@ struct RGBColor {
 };
 
 typedef xt::xtensor<RGBColor, 2> ImageData;
-typedef xt::xtensor<int32_t, 3>  IdData;
+
+struct IdData
+{
+  // Constructor
+  IdData(int h_res, int v_res);
+
+  // Methods
+  void set_value(int y, int x, const Particle& p, int level);
+
+  // Members
+  xt::xtensor<int32_t, 3> data_; //!< 2D array of cell & material ids
```

Comment:
A little confusing to say "2D array" when it has three dimensions

Reply:
True true. I was struggling with how to describe these because of that last dimension.

How about: "Array of cell & material Ids with dimensions (h_res, v_res, 2) where 0 - cell_id, 1, material_id"

That kind of doc string makes me think that we should just define these indices somewhere, maybe in the respective `ID`/`Property` structs? Thoughts?